sdk/core/Azure.Core/src/Pipeline/ContentTypeUtilities.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+
+namespace Azure.Core.Pipeline
+{
+    internal class ContentTypeUtilities
+    {
+        public static bool IsText(string contentType, out Encoding encoding)
```

Comment:
Maybe something like `TryGetTextEncoding` would be fit better?   Having an `out` param with an `IsXXX` name feels weird.